skt/__init__.py

```diff
@@ -497,11 +520,19 @@ def __init__(self, path, basecfg, cfgtype=None, makeopts=None,
 
     def prepare(self, clean=True):
         if (clean):
-            args = self.defmakeargs + ["mrproper"]
+            args = self.makeopts + ["mrproper"]
             logging.info("cleaning up tree: %s", args)
             subprocess.check_call(args)
 
-        shutil.copyfile(self.basecfg, "%s/.config" % self.path)
+        if self.kbuild_output_dir:
+            config_destination = "%s/.config" % self.kbuild_output_dir
+        else:
+            config_destination = "%s/.config" % self.path
```

Comment:
Any reason why we are not using below's get_cfgpath()?

Reply:
Oh good point - I missed that. That would simplify things.